### PR TITLE
Email signin

### PR DESCRIPTION
**Pull request by alxdarksage**

### CODE REVIEW

**Comment by alxdarksage** [comment]
Looks good.

File: _data/versions.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
-java_sdk: 0.12.43
+java_sdk: 0.12.47
```
Comment (DwayneJengSage) [inline _data/versions.yml]: What is this used for? Should it match the version in JavaSDK (which will need to be bumped to 12.49 or 12.50)?

**Review by DwayneJengSage — COMMENTED** [review]
One minor comment, everything else looks good.

File: _data/versions.yml (hunk 1 above)
Comment (alxdarksage) [reply _data/versions.yml]: It's used to show the correct SDK number in the documentation, and to link to that version's javadocs. Every version's javadocs are posted to S3, so it's okay if this lags behind the most recent version... but as features get introduced, we want to point the documentation toward newer versions of the SDK. So periodically I bump this up.

**Pull request merged**